src/Terminal/determineAllPossibilitiesForTabCompletion.ts

```diff
@@ -204,7 +204,7 @@ export async function determineAllPossibilitiesForTabCompletion(
 
   if (isCommand("buy")) {
     const options = [];
-    for (const i in DarkWebItems) {
+    for (const i of Object.keys(DarkWebItems)) {
```

Comment:
Good idea, but work left to be done.

For instance this should be:

```js
for (const item of DarkWebItems) {
  options.push(item.program)
}
```

Reply:
`DarkWebItems` is not an array, rather a mapping, and cannot be iterated in that way. Save for a few instances, all changes are to iterations of map-like objects.